p2panda-rs/src/schema/schema.rs

```diff
@@ -575,3 +646,46 @@ mod tests {
         .is_err());
     }
 }
+// Exclude wasm which doesn't support `Node`
+#[cfg(not(target_arch = "wasm32"))]
+#[cfg(test)]
+mod tests_wasm_not_invited {
```

Comment:
In the refactor I'll move this in another place as this is already much more of an "integration" test, I will write some more unit tests here instead :+1: